### PR TITLE
Newrelic monitoring

### PR DESCRIPTION
**Pull request by mklauber**

### CODE REVIEW

File: seo/templatetags/seo_extras.py [hunk 1]
```diff
@@ -24,6 +25,12 @@
 register = template.Library()
 
 
+@register.inclusion_tag('includes/newrelic_browser_monitoring.html')
+def newrelic_browser_monitoring():
+    # Only use the script tag on 1% of page loads.
+    return {'enabed': random.random() < 0.01 }
```
Comment (JLMcLaugh) [inline seo/templatetags/seo_extras.py]: Typo on enabled.

Comment (JLMcLaugh) [inline seo/templatetags/seo_extras.py]: While we're at it. Could you name it something that's more meaningful when read from the temeplate? newrelic_enabled or something similar.

Comment (tdphillips) [inline seo/templatetags/seo_extras.py]: Ping

Comment (mklauber) [inline seo/templatetags/seo_extras.py]: Fixed.

File: templates/includes/newrelic_browser_monitoring.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
...),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(0>o?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=n},{}],24:[function(t,e){function n(t){return!(t&&"function"==typeof t&&t.apply&&!t[i])}var r=t("ee"),o=t(1),i="nr@original",a=Object.prototype.hasOwnProperty;e.exports=function(t){function e(t,e,r,a){function nrWrapper(){var n,i,s,f;try{i=this,n=o(arguments),s=r&&r(n,i)||{}}catch(d){u([d,"",[n,i,a],s])}c(e+"start",[n,i,a],s);try{return f=t.apply(i,n)}catch(p){throw c(e+"err",[n,i,p],s),p}finally{c(e+"end",[n,i,f],s)}}return n(t)?t:(e||(e=""),nrWrapper[i]=t,f(t,nrWrapper),nrWrapper)}function s(t,r,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<r.length;c++)s=r[c],a=t[s],n(a)||(t[s]=e(a,f?s+o:o,i,s))}function c(e,n,r){try{t.emit(e,n,r)}catch(o){u([o,e,n,r])}}function f(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){u([r])}for(var o in t)a.call(t,o)&&(e[o]=t[o]);return e}function u(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=r),e.inPlace=s,e.flag=i,e}},{1:23,ee:"QJf3ax"}]},{},["G9z0Bl",4,11,5]);
```
Comment (aspidites) [inline templates/includes/newrelic_browser_monitoring.html]: I thought we said this was going in an external static file?

Comment (mklauber) [inline templates/includes/newrelic_browser_monitoring.html]: Putting it in an external file doesn't achieve the goal of this file.  As this is for monitoring, loading it asynchronously means we aren't able to monitor other asynchronous calls.  We wouldn't know when they were started.

**Pull request merged**